src/sonic-yang-models/yang-models/sonic-flex_counter.yang

```diff
@@ -119,6 +149,12 @@ module sonic-flex_counter {
                 leaf FLEX_COUNTER_DELAY_STATUS {
                     type flex_delay_status;
                 }
+                leaf POLL_INTERVAL {
```

Comment:
If it stayed in same range and with same default value, then we can have typedef similar as Example:flex_delay_status.

Reply:
But we have three different ranges and three different default values for POLL_INTERVAL for different containers.

Reply:
@praveen-li 
I added a typedef for poll_interval range 1000..30000 and default 10000.
Comments from other reviewers have been resolved.
Please review.

Reply:
@praveen-li I added a typedef for all poll_interval ranges. Please review.